src/App.tsx

```diff
@@ -1,9 +1,23 @@
-import React from 'react';
+import React, { useState } from 'react';
 import './App.scss';
 import { MoviesList } from './components/MoviesList';
 import moviesFromServer from './api/movies.json';
 
 export const App: React.FC = () => {
+  const [query, setQuery] = useState('');
+
+  const filterMovies = moviesFromServer.filter(movie => {
+    const lowerQuery = query.toLowerCase().trim();
+    const lowerTitle = movie.title.toLowerCase().trim();
+    const lowerDescription = movie.description.toLowerCase().trim();
```

Comment:
you can create separate function for converting to lower case + check if query included somewhere